jsm.go

```diff
@@ -26,49 +26,49 @@ import (
 // JetStreamManager is the public interface for managing JetStream streams & consumers.
 type JetStreamManager interface {
 	// AddStream creates a stream.
-	AddStream(cfg *StreamConfig) (*StreamInfo, error)
+	AddStream(cfg *StreamConfig, opts ...JSOpt) (*StreamInfo, error)
```

Comment:
AddStream is getting a panic here: https://github.com/nats-io/nats.go/blob/7ea5697e139f81d56e991d486b07cec9e8199858/test/js_test.go#L106-L116

Reply:
maybe the error is not being returned some how after the attempts?

Reply:
Ooops, that's my bad. I purposely added some bad code to make it fail. I forgot to take that out! 😄